File: ApiSuvesaPos/Negocio/Modelo/cxp/ajustescpagar/ajustescpagar.vb
Namespace Modelo
    Public Class ajustescpagar
        Public Property ID_Ajuste As Long
        Public Property AjusteNo As Long
        Public Property Tipo As Char
        Public Property Cod_Proveedor As Integer
        Public Property Nombre_Proveedor As String
        Public Property Fecha As DateTime
        Public Property Saldo_Prev As Decimal
        Public Property Monto As Decimal
        Public Property Saldo_Act As Decimal
        Public Property Observaciones As String
        Public Property Anula As Boolean
        Public Property Cod_Usuario As String
        Public Property Contabilizado As Boolean
        Public Property Cod_Moneda As Integer
        Public Property Asiento As Long
        Public Property FechaEntrada As DateTime

    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxp/ajustescpagar/ajustescpagar.cs
using System;

namespace Modelo
{
    public class ajustescpagar
    {
        public long ID_Ajuste { get; set; }
        public long AjusteNo { get; set; }
        public char Tipo { get; set; }
        public int Cod_Proveedor { get; set; }
        public string Nombre_Proveedor { get; set; }
        public DateTime Fecha { get; set; }
        public decimal Saldo_Prev { get; set; }
        public decimal Monto { get; set; }
        public decimal Saldo_Act { get; set; }
        public string Observaciones { get; set; }
        public bool Anula { get; set; }
        public string Cod_Usuario { get; set; }
        public bool Contabilizado { get; set; }
        public int Cod_Moneda { get; set; }
        public long Asiento { get; set; }
        public DateTime FechaEntrada { get; set; }
    }
}

